src/client/containers/BoardPageComponents/BoardAddCandidate/BoardHeader.style.css

```diff
@@ -0,0 +1,15 @@
+* {
```

Comment:
This * will affect all styles in the app

Reply:
Done and checked in storybook it didn't impact on other components.